Лабораторная работа 3 (объектно-ориентированное программирование)/task_1.py

```diff
@@ -0,0 +1,103 @@
+class Book:
+
+    def __init__(self, name: str, author: str):
+        self._name = None
+        self._author = None
+        self.name = name
+        self.author = author
```

Comment:
```suggestion
        self._name = name
        self._author = author
```
При таком присваивании значений защищенным переменным вы как рза накладываете на них свойства написанные далее. Свойства это своего рода альтернатива методам, но они вызываются только при изменении переменных. 
Запись которая у вас сейчас не корректна - так как вы защищенным переменным присвоили значение None, а потом переменным экземпляра напрямую задали введенные значения